test/integration/server.test.js

```diff
@@ -114,19 +115,39 @@ test('GET /getSimulationReport', async (t) => {
 });
 */
 
-// TODO: create generateSimulationReport test
-
 test('GET /getSimulationReportList', async (t) => {
     const response = await fastify.inject({
         method: 'GET',
         url: '/getSimulationReportList'
     });
     assert.strictEqual(response.statusCode, 200);
-    const payloadList = response.payload;
+    const payloadList = JSON.parse(response.payload);
     if (!payloadList) console.error("payload is null");
     else console.log("All simulation reports: ", payloadList);
+
+    // Predefined globals for testing..
+    testPrintJobId = payloadList[0].PrintJobID;
```

Comment:
Instead of these globals, just have the generateSimulationReport test call getSimulationReport so it has the IDs